lib/index.js

```diff
@@ -217,6 +233,9 @@ export function Markdown(options) {
 
   return toJsxRuntime(hastTree, {
     Fragment,
+    // @ts-expect-error
+    // React components are allowed to return numbers,
+    // but not according to the types in hast-util-to-jsx-runtime
```

Comment:
Is this related to https://github.com/syntax-tree/hast-util-to-jsx-runtime/pull/6 ?

Reply:
Somewhat yes. But also apparently `hast-util-to-jsx-runtime` doesn’t support `JSX.ElementType`, which JSX runtimes may optionally define. That adds another complexity on top of that issue.